README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying Kubernetes/OpenShift cluster
+
+A Kafka/Zookeeper cluster can be deployed on a Kubernete or OpenShift cluster. So, first of all, such cluster is needed.
```

Comment:
There are two clusters in this sentence, so "first of all, such cluster is needed" is rather ambiguous. How about:

> You will need a Kubernetes or OpenShift cluster to deploy Barnabas.